articles/110_interface_definition.md

```diff
@@ -70,11 +70,15 @@ The following primitive types are defined:
 * `string`
 
 <div class="alert alert-warning" markdown="1">
+  **Question:** in ROS 1, `string` was ASCII. The Python 3 `str` type is UTF-8. What is the DDS `string` type?
+</div>
+
+<div class="alert alert-warning" markdown="1">
   <b>TODO:</b> consider <code>wchar</code>, <code>wstring</code>, <code>u16string</code>, <code>u32string</code>
 </div>
 
 <div class="alert alert-warning" markdown="1">
-  <b>TODO:</b> consider removing <code>byte</code>, <code>char</code> after specifying the mapping to C++ and Python
```

Comment:
I went ahead with this change. Those types have been deprecated for many years, and the mapping was already clearly specified:

* `char`: deprecated alias for `uint8`
* `byte`: deprecated alias for `int8`

See: http://wiki.ros.org/msg#Fields